### ISSUE TITLE
sink message of debezium json without schema field

### ISSUE THREAD

**Issue by masterclock** [risingwavelabs/risingwave#11539]
### Is your feature request related to a problem? Please describe.

when create a sink like:
```
CREATE SINK some_sink FROM some_view WITH (
    connector = 'kafka',
    properties.bootstrap.server = 'localhost:9092',
    type = 'debezium',
    topic = 'cloud_testees_with_station',
    primary_key = 'id'
);
```
the output message looks like:
```
{
  "schema": ...,
  "payload": ""
}
```
as the "schema" part in message is very large and rarely used, I'd like to remove this field, just like configuration in debezium
```
schamas.enable = false
```

### Describe the solution you'd like

a configuration in `WITH` to disable schema generation.

**Comment by fuyufjh** [comment]
I am not sure whether the debezium format definition allows it 🤔 @tabVersion can you please take a look?

**Comment by tabVersion** [comment]
> I am not sure whether the debezium format definition allows it 🤔 @tabVersion can you please take a look?

yes, debezium allows this operation by specifying `key.converter.schemas.enable` and `value.converter.schemas.enable`.
I am not sure if we can do this now because the generation logic is nested into kafka sink & kinesis sink. Maybe some prior refactoring is needed.

**Comment by devillove084** [comment]
@tabVersion Hi, is there any progress on this issue, I'd like to try and help in any way I can if I can.

**Comment by tabVersion** [comment]
> @tabVersion Hi, is there any progress on this issue, I'd like to try and help in any way I can if I can.

Sorry, there's no update yet. And contribution is welcomed

**Comment by xiangjinwu** [comment]
We already introduced `schemas.enable` for upsert json sink in #12113, which is essentially the same option from debezium because both are based on the Java class `org.apache.kafka.connect.json.JsonConverter`. There has also been some recent fixes regarding how RisingWave types maps to these "kafka connect types" by @xuefengze.

Moving forward, I would suggest to:
* Refactor `DebeziumJsonFormatter` to leverage the `kafka_connect` option from `JsonEncoder`.
* Use `schemas.enable` as the user facing name. When absent, default to `false` for `upsert` but `true` for `debezium`. This is necessary for backward compatibility.

**Comment by cnsky1103** [comment]
hi I've submitted a PR to solve this, @tabVersion can you do some code review?
